plugins/nvme/fu-nvme-device.c

```diff
@@ -0,0 +1,330 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fcntl.h>
+#include <string.h>
+#include <sys/errno.h>
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include <glib/gstdio.h>
+
+#include <linux/nvme_ioctl.h>
+
+#include "dfu-chunked.h"
+
+#include "fu-nvme-device.h"
```

Comment:
I think you want to re-order these headers, wasn't the convention we were going for to have "local" first followed by system?

Reply:
I think it's config.h, system, local.

Reply:
Hmm, I guess that matches what we discussed previously (https://github.com/hughsie/fwupd/pull/568#pullrequestreview-130516573) I just remembered backwards.